src/layers/core/geojson-layer/geojson-layer.js

```diff
@@ -102,16 +101,17 @@ export default class GeoJsonLayer extends CompositeLayer {
       getElevation,
       getColor: getFillColor,
       updateTriggers: {
-        getElevation: this.props.updateTriggers.getElevation,
-        getColor: this.props.updateTriggers.getFillColor
+        getElevation: updateTriggers.getElevation,
+        getColor: updateTriggers.getFillColor
       },
       onHover,
       onClick
     }));
 
     // Polygon outline or wireframe
     let polygonOutlineLayer = null;
-    if (drawPolygons && extruded && wireframe) {
+    // If extruded, check the "wireframe" prop
+    if (extruded && wireframe && drawPolygons) {
       polygonOutlineLayer = new SolidPolygonLayer(Object.assign({}, this.props, {
```

Comment:
Like in PolygonLayer below, recommend changing this variable to `polygonWireframeLayer` as outlines and wireframes are different concepts.